fbw-a380x/src/wasm/systems/a380_systems/src/electrical/mod.rs

```diff
@@ -58,9 +58,11 @@ impl A380Electrical {
     const MIN_EMERGENCY_GENERATOR_RPM_TO_ALLOW_CURRENT_SUPPLY: f64 = 2000.;
 
     const RAT_CONTROL_SOLENOID1_POWER_BUS: ElectricalBusType =
-        ElectricalBusType::DirectCurrentHot(1);
+        ElectricalBusType::DirectCurrentEssential;
```

Comment:
```suggestion
        ElectricalBusType::DirectCurrentHot(1);
```
This was correct before

Reply:
You're the last one to check electrical stuff so I believe you :D